src/jsx.re

```diff
@@ -0,0 +1,17 @@
+type element;
+type ref;
+
+[@bs.val] external null: element = "null";
+
+external float: float => element = "%identity";
+external int: int => element = "%identity";
+external string: string => element = "%identity";
+
+external array: array(element) => element = "%identity";
```

Comment:
You might need to `include Jsx` in React module, otherwise those functions won't be available under `React.*`